addons/web_tour/static/src/tour_service/tour_helpers.js

```diff
@@ -263,6 +263,35 @@ export class TourHelpers {
         hoot.select(values, { target: element });
     }
 
+    /**
+     * Ensures that the given {@link Selector} is unchecked.
+     * @description
+     * If it is checked, a click is triggered on the input.
+     * If the input is still checked after the click, an error is thrown.
+     *
+     * @param {string|Node} selector
+     * @example
+     *  run: "uncheck", // Unchecks the action element
+     * @example
+     *  run: "uncheck input[type=checkbox]", // Unchecks the selector
+     */
+    uncheck(selector) {
+        const element = this._get_action_element(selector);
+        this._ensureEnabled(element, "uncheck");
+        hoot.uncheck(element);
+    }
+
+    /**
+     * Navigate to {@link url}.
+     *
+     * @param {string} url
+     * @example
+     *  run: "goToUrl /shop", // Go to /shop
+     */
+    goToUrl(url) {
+        window.location = url;
+    }
```

Comment:
Hello, you should create an <a> element, set its href and then click on it. 
This is for secutiry reasons: 
- Compliance with Security Policies (Content Security Policy - CSP)
- Less prone to XSS attacks
- Less prone to timing errors

```javascript
const link = document.createElement('a');
link.href = url;
document.body.appendChild(link);
link.click();
```

Reply:
I've pushed the change without `document.body.appendChild(link);`, because it works without it. Is there any particular reason we should keep it?

Reply:
@pipu-odoo (@mart-e since you seem to agree ?)

> Compliance with Security Policies (Content Security Policy - CSP)

How is that a factor here? We are in a tour, we want the equivalent of the user typing that URL into its browser. If the URL is reachable that way, that's what we want.

> Less prone to XSS attacks

1. How is that less prone to XSS attacks using a link?
2. Why is it even a factor when it comes to writing a tour?

> Less prone to timing errors

What's that?

Reply:
Since we are in the towers, I let Security take a stand. Here, I am only sharing "good practices" regarding redirection that Security has already shared with me ... even in the case of towers. As for "Less prone to timing errors", this concerns the management of DOMContentLoaded, beforeUnload, load, etc. events.